downloadDirectory/README.md

```diff
@@ -186,6 +186,16 @@ This step copies the `gene_association.reactome` file that is produced during th
 
 This step copies the `models2pathways.tsv` file that is produced during the <a href="https://github.com/reactome/Release/tree/master/scripts/release/biomodels">Biomodels</a> step. Comparing the file to previous releases is sufficient for this step.
 
+<h4>Protege exporter</h4>
+This step will run the protege export code in `GKB::WebUtils`. The Perl code will create an archive file containing a pins, pont, and pprj file, for a given pathway. This code is designed to only export TOP-LEVEL pathways, as defined as pathways associated with the FrontPageItem in the database.
+
+This step takes the following configuration options:
+ - protegeexporter.pathToWrapperScript - This is the _absolute_ path to the directory containing the Perl wrapper script `run_protege_exporter.pl`. The script should be located in `src/main/resources` for _this_ project.
+ - protegeexporter.parallelism - The number of concurrent protege export jobs to run. Try to keep this smaller than the number of available cores (MySQL and your operating system should get 1 core each, at least).
```

Comment:
Just to clarify, does this mean the number of cores should always be set to at least 2?

Reply:
It's probably a good idea to do that, if you can. If you don't set anything, then it defaults to whatever ForkJoinPool defaults to on that system (usually: number_of_cores - 1)

Reply:
Got it!